pymc3/distributions/distribution.py

```diff
@@ -88,8 +88,11 @@ def getattr_value(self, val):
         return val
 
     def _repr_latex_(self, name=None, dist=None):
```

Comment:
I think this is less code, right?  plus it will continue to inherit the signature of `_repr_latex_` with no work